package/yast2-vm.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Thu May 10 13:43:20 MDT 2018 - carnold@suse.com
+
+- bsc#1091967 - relocation-server configuration - error : cannot
+  read the current libvirt/sshd state
```

Comment:
Version is missing, and also a version bump.
`rake version:bump`

We do plan to release it as part of GA - media, then the SLE-15-GA branch should be used instead of master